index.html

```diff
@@ -5,10 +5,183 @@
+        <hr class="features__line">
+        <p class="features__body">
+          Lorem ipsum dolor sit amet, consectetur adipisc 
+          Pellentesque vel enim a elit viverra elementuma.
+          Aliquam erat volutpat.  
+        </p>
+      </article>
+      <article class="features__item">
+        <img class="features__img" src="img/social.png" alt="get social">
+        <h3 class="features__header">GET SOCIAL</h3>
+        <hr class="features__line">
+        <p class="features__body">
+          Lorem ipsum dolor sit amet, consectetur adipisc 
+          Pellentesque vel enim a elit viverra elementuma.
+          Aliquam erat volutpat.            
+        </p>
+      </article>
+    </section>
+
+    <section class="app">
+      <h2 class="h2_visible">INTRODUCING THE FUDI APP</h2>
+      <img class="app__img" src="img/iPhone.png" alt="FUDI APP">
+      <article class="app__article">
+        <h3 class="app__header">INTRODUCING THE FUDI APP</h3>
+        <hr class="app__line">
+        <p class="features__body app__body">
+          Morbi eget posuere dolor. Pellentesque cursus aliquet aliquet. Aeneanet 
+          felis sit amet diam mollis ullamcorper. Nullam consequat sem a ante vest
+          ibulum tristique. Suspendisse tristique lacus ac mattis porta. 
+        </p>
+        <p class="features__body app__body">
+          Vivamus ligula quam, vehicula non lacinia sed, faucibus sit amet libero. In 
+          libero dui, eleifend eu nisi id, egestas fringilla odio. In varius quam a mas
+          sa hendrerit ullamcorper a eu justo. Suspendisse porta mattis convallis. 
+          Aenean tempus ligula ac odio rhoncus, quis aliquam dolor accumsan. 
+        </p>
+        <p class="features__body app__body">
+          Suspendisse aliquet felis consectetur libero congue, sed pulvinar diam 
+          malesuada. Duis vehicula a nibh id hendrerit. Donec sit amet ultrices 
+          ante, a mattis massa
+        </p>
+        <button class="app__but">
+          <img class="butt__img" src="img/but.png" alt="App Store">
+        </button>
+        <button class="app__but">
+          <img class="butt__img" src="img/but1.png" alt="Google play">
+        </button>
+      </article>
+    </section>
+
+    <section class="comments">
+      <h2 class="h2_visible">Review</h2>
+      <img class="coments__img" src="img/review.png" alt="review">
+      <p class="comments__body">
+        “I am so happy because I found this recipe, and it just made my life easier.<br>
+        Thanks  so much for sharing!”<br>
+        <span class="comments__writer">- Michael Dawson, San Francisco, CA -</span>
+      </p>
+      <ul class="comments__list">
+        <li class="comments__list_item"><img src="img/circle_white.png" alt="circle"></li>
+        <li class="comments__list_item"><img src="img/circle.png" alt="circle"</li>
+        <li class="comments__list_item"><img src="img/circle.png" alt="circle"</li>
```

Comment:
you forgot to close img tag here. it is better to validate it https://validator.w3.org/